src/platform/silabs/wifi_args.gni

```diff
@@ -40,7 +40,7 @@ lwip_root = "${efr32_sdk_build_root}/silabs_lwip"
 #lwip_platform = "external"
 lwip_platform = "silabs"
 lwip_ipv6 = true
-lwip_ipv4 = false
+lwip_ipv4 = true
```

Comment:
why this is enabled?

Reply:
As this is enabled in siwx917_sdk.gni through build params, so not required here.
Thanks for the comment @chirag-silabs